src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -503,102 +428,85 @@ RoverPositionControl::task_main()
 	orb_unsubscribe(_pos_sp_triplet_sub);
 	orb_unsubscribe(_vehicle_attitude_sub);
 
-	_task_running = false;
-
 	warnx("exiting.\n");
```

Comment:
fyi you can remove that as well, and `warnx` is deprecated, `PX4_{WARN,INFO,ERR}` is used instead.